webdriver/tests/bidi/network/auth_required/auth_required.py

```diff
@@ -15,21 +17,16 @@ async def test_subscribe_status(
     async def on_event(method, data):
         events.append(data)
 
-    remove_listener = bidi_session.add_event_listener(AUTH_REQUIRED_EVENT, on_event)
+    remove_listener = bidi_session.add_event_listener(
+        AUTH_REQUIRED_EVENT, on_event)
 
     auth_url = url(
         "/webdriver/tests/support/http_handlers/authentication.py?realm=testrealm"
     )
 
     on_auth_required = wait_for_event(AUTH_REQUIRED_EVENT)
 
-    # navigate using wait="none" as other wait conditions would hang because of
-    # the authentication prompt.
-    await bidi_session.browsing_context.navigate(
-        context=new_tab["context"],
-        url=auth_url,
-        wait="none",
-    )
+    asyncio.ensure_future(fetch(url=auth_url, context=new_tab))
```

Comment:
Can we alternatively solve it by not blocking on the navigate call? and rely on the event instead